src/extensions/default/bramble/stylesheets/sidebarTheme.css

```diff
@@ -1,68 +1,250 @@
 }
 span#project-title.title {
-	display: none;
+  display: none;
 }
 div.working-set-option-btn[style] {
-	display: none !important;
+  display: none !important;
 }
 
-/* Recoloring */
+/* Dark Theme Color */
+/* TODO - move this to the dark theme CSS later */
+
 #sidebar {
-	background-color: black;
-	text-shadow: none;
-	color: #333333;
+  background-color: black;
+  text-shadow: none;
+}
+
+#project-files-container {
+  padding-left: 0px;
+  font-weight: 400;
+  font-family: "Open Sans", "Helvetica Neue";
+}
+
+#project-files-container ul {
+  padding: 0;
+}
+
+
+/* File tree basic styles */
+
+li.jstree-leaf a span,
+li.jstree-leaf span.extension {
+  font-size: 15px;
+  color: rgba(255,255,255,.5);
+}
+
+li.jstree-closed > a span,
+li.jstree-open > a span {
+  font-size: 15px;
+  color: rgba(255,255,255,.7);
+}
+
+
+/* Overrides not needed margin */
+
+.jstree-brackets li.jstree-leaf,
+.jstree-brackets li.jstree-closed,
+.jstree-brackets li.jstree-open {
+  margin-left: 0;
+}
+
+
+/* Basic formatting of all items in file menu */
+
+li.jstree-open a,
+li.jstree-closed a,
+li.jstree-leaf a {
+  padding: 10px 10px 10px 30px;
+  cursor: pointer !important;
 }
-li.jstree-leaf>a {
-	padding: 5px;
+
+
+/* Folders get bold text treatment */
+
+li.jstree-open > a,
+li.jstree-closed > a {
+  font-weight: 600;
+}
+
+
+/* Selected Item */
+
+li.jstree-leaf a.selected-node,
+li.jstree-leaf a.selected-node span,
+li.jstree-leaf a.selected-node .extension {
+  color: white !important;
+}
+
+
+/* Hover state for all items */
+
+li.jstree-leaf a:not(.selected-node):hover,
+li.jstree-open a:first-of-type:hover,
+li.jstree-closed a:first-of-type:hover
+{
+  background: rgba(255,255,255,.08);
+}
+
+
+/* Hover state item text */
+
+li.jstree-leaf a:not(.selected-node):hover > span
+{
+  color: rgba(255,255,255,.6);
+}
+
+li.jstree-open > a span
+li.jstree-closed > a span,
+{
+  color: rgba(255,255,255,.7);
+}
+
+
+/* Message for when there are no files in a folder */
+
+li.jstree-open > ul:empty:after {
+  content: "This folder is empty";
+  color: rgba(255,255,255,.4);
+  padding: 10px 10px 10px 30px;
+  font-style: italic;
+  display: block;
 }
-li.jstree-leaf>a>span {
-	font-size: 15px;
-	color: #666666;
+
+
+/* Right-clicked file */
+
+li.jstree-closed a.context-node,
+li.jstree-open a.context-node,
+li.jstree-leaf a.context-node {
+  background: rgba(255,255,255,0.08);
 }
-li.jstree-open>a>span {
-	font-size: 15px;
-	color: #666666;
+
+
+/* Currently selected (edited) file */
+
+li.jstree-leaf a.selected-node,
+li.jstree-leaf a.selected-node:hover {
+  background: #222;
 }
-li.jstree-closed>a>span {
-	font-size: 15px;
-	color: #666666;
+
+
+/* Adds icon for folders only icon */
+
+.jstree-brackets li.jstree-closed > .jstree-icon,
+.jstree-brackets li.jstree-open > .jstree-icon {
+  transition: transform .15s ease-out !important;
+  top: 9px;
+  left: 7px !important;
+  margin: 0;
+  background-image: url(/src/styles/images/folder-icon.svg);
```

Comment:
This will not work on staging/prod because there is no `src` dir there.

If you're doing a cleanup, would you be able to move out all the images from `src/extensions/default/bramble/lib` into an images folder in maybe `src/extensions/default/bramble/images`. You can then put this image into that folder and reference it here as `../images/folder-icon.svg` and it'll work for local dev as well as staging and prod.